test/support/console_test_case.rb

```diff
@@ -306,54 +181,62 @@ def manual_debug_code(program)
       write_temp_file(strip_line_num(program))
       remote_info = setup_unix_domain_socket_remote_debuggee
 
-      Timeout.timeout(TIMEOUT_SEC) do
```

Comment:
Can we remove this timeout? This looks like it could cause an infinite loop.

Reply:
Won't the timeout prevent that from happening instead? 

Or did you mean **not** removing it? I now see it's removed unintentionally.

Reply:
Yeah I meant whether or not it's ok to remove the timeout.

Reply:
Ah right. I didn't realize it was a removal so I thought you were asking me to remove it 😬